### ISSUE TITLE
Question / Potential Bug re: Seq2Seq Example 

### ISSUE THREAD

**Issue by JosephGatto** [THU-KEG/OmniEvent#41]
For the evaluation code provided in https://github.com/THU-KEG/OmniEvent/blob/main/examples/EAE/seq2seq.py 

```
logits, labels, metrics, test_dataset = predict(trainer=trainer, tokenizer=tokenizer, data_class=data_class,
                                                        data_args=data_args, data_file=data_args.test_file,
                                                        training_args=training_args)
  preds = get_pred_s2s(logits, tokenizer, pred_types=training_args.data_for_evaluation["pred_types"])

  logging.info("\n")
  logging.info("{}-EAE Evaluate Mode : {}-{}".format("-" * 25, data_args.eae_eval_mode, "-" * 25))
  logging.info("{}-Use Golden Trigger: {}-{}".format("-" * 25, data_args.golden_trigger, "-" * 25))

  if data_args.test_exists_labels:
      logging.info("{} test performance before converting: {}".format(data_args.dataset_name, metrics))
      get_ace2005_argument_extraction_s2s(preds, labels, data_args.test_file, data_args, None)
```

It seems that the labels being passed to get_ace2005_argument_extraction_s2s are still token ids, but the function is expecting it to have been parsed and prepared similar to how preds is formatted. Is there missing code here? 

Thanks! 

Note I am adapting this code for RAMs and using t5-base config.

**Comment by h-peng17** [comment]
Thanks for the question. In function ``get_ace2005_argument_extraction_s2s ``, the parameter `labels` is useless, and we convert the sequence prediction `preds` into token classification format, see https://github.com/THU-KEG/OmniEvent/blob/main/OmniEvent/evaluation/README.md.
